apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -877,38 +872,55 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
                                     ) }
                                 { !isLoading && !hideUserStoreDropdown()
                                     && (
-                                        <Grid.Row columns={ 1 }>
-                                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 10 }>
-                                                <Form.Field required={ true }>
-                                                    <label className="pb-2">
-                                                        { t("console:manage.features.user.forms.addUserForm.inputs."+
-                                            "domain.placeholder") }
-                                                    </label>
-                                                    <Dropdown
-                                                        className="mt-2"
-                                                        fluid
-                                                        selection
-                                                        labeled
-                                                        options={ readWriteUserStoresList }
-                                                        loading={ isLoading }
-                                                        data-testid={ `${componentId}-userstore-dropdown` }
-                                                        data-componentid={ `${componentId}-userstore-dropdown` }
-                                                        name="userstore"
-                                                        disabled={ false }
-                                                        value={ selectedUserStore }
-                                                        onChange={
-                                                            (e: React.ChangeEvent<HTMLInputElement>,
-                                                                data: DropdownProps) => {
-                                                                setSelectedUserStore(data.value.toString());
-                                                                setSelectedUserStore(data.value.toString());
+                                        <>
```

Comment:
Remove this unnecessary component.

Reply:
Since there are two `Grid.Rows` this component is required.